EC-84: Restructure eZ Commerce doc, part 2 (#2)
ezsystems/commerce-documentation: docs/guide/basket/basket_features/discontinued_products.md

# Discontinued products

> A listener can check if a product is not sold anymore (discontinued) and the user might have to be informed that this product is not available any more. This feature can be disabled (`discontinued_products_listener_active`).

Reviewer: "and the user might have to be informed that this product is not available any more." -> "so that the user is informed that the product is not available"?
"This feature can be disabled (`discontinued_products_listener_active`)." -> how? Would it be:
"This feature can be disabled by toggling the `discontinued_products_listener_active` setting."?

> A listener can check if a product is not sold anymore (discontinued) and the user might have to be informed that this product is not available any more. This feature can be disabled (`discontinued_products_listener_active`).

Reviewer 2: ```suggestion
A listener can check if a product is not sold anymore (`discontinued`) so the user can be informed that this product is not available any more. This feature can be disabled (`discontinued_products_listener_active`).
```

> The listener will check if the current stock is equal or greater than the quantity the customer wants to order. In this case the order will be allowed. The optional config setting `discontinued_products_listener_consider_packaging_unit` might enable skipping the packaging unit in order to sell the remaining products even if remaining stock does not fit to the packing unit rule (e.g. packing unit = 10 pieces but 9 are left in stock). The listener will reduce the quantity in the order to the number of products which are in stock. 

Reviewer: I would definitely suggest using present tense.
'might enable"-> under what circumstances? -> "enables"
